lib/twingly/livefeed/parser.rb

```diff
@@ -44,12 +44,16 @@ def create_result(data_node)
       def parse_post(element)
         post_params = {}
         element.element_children.each do |child|
-          if %w(tags links images).include?(child.name)
-            post_params[child.name] = parse_array(child)
-          else
-            post_params[child.name] = child.text
-          end
+          post_params[child.name] =
+            if %w(tags links images).include?(child.name)
+              parse_array(child)
+            elsif child.name == "coordinates"
+              parse_coordinates(child)
+            else
+              child.text
+            end
```

Comment:
:+1: I had the same thought as well, but I started to do something else and forgot about it 😄